data/en/invoke.json

```diff
@@ -25,6 +25,24 @@
 			"description":"Invokes the size method on a new HashMap object, which should return 0",
 			"code":"invoke(createObject(\"java\", \"java.util.HashMap\"), \"size\")",
 			"result":"0"
+		},
+		{
+			"title":"Invoke a method on a component",
```

Comment:
I would add "(script syntax)" to the second and "(tag syntax)" to the first example for clarity.